packages/signals/sig.pony

```diff
@@ -199,9 +199,10 @@ primitive Sig
     end
 
   fun usr2(): U32 =>
-    ifdef bsd or osx then 31
-    elseif linux then 12
-    else compile_error "no SIGUSR2"
+    ifdef linux or bsd or osx then
+      compile_error "SIGUSR2 reserved for runtime use"
+    else
+      compile_error "no SIGUSR1"
```

Comment:
Shouldn't this still be `SIGUSR2`?

Reply:
Yes. I'll fix this when I rebase and fix conflicts.